### PR TITLE
GIVCAMP-266 | Add dark overlay options; rich text story caption; smaller WYSIWYG headings

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Add dark overlay options for Story MVP Heroes and Blurry Posters
- Step down regular WYSIWYG heading sizes by one modular step
- Change story hero caption to rich text
- Other misc updates

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. https://deploy-preview-174--giving-campaign.netlify.app/
2. Go to the homepage and look at the blurry posters (featured story horizontal posters). See that the dark overlay in the background image works.
3. Check on mobile that things look ok
4. Go to a few different story and check the story hero dark overlay
https://deploy-preview-174--giving-campaign.netlify.app/stories/the-most-wicked-problems
https://deploy-preview-174--giving-campaign.netlify.app/stories/video-reading-the-room
https://deploy-preview-174--giving-campaign.netlify.app/stories/from-human-health-to-climate-health
5: Note: some of the hero images at large XS to SM breakpoint might get chopped off in a weird way - I'll tweak that in another PR
6. Go to this story
https://deploy-preview-174--giving-campaign.netlify.app/stories/using-machine-learning-to-help-refugees-succeed
Check that h2 within the story content WYSIWYG is now type-3 (used to be type-4)
Screenshot 2023-12-15 at 2 36 29 PM
7. Go to https://deploy-preview-174--giving-campaign.netlify.app/stories/cardinal-rules
8. Check that the hero caption under the hero is now rich text (has italic text)

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-266

### CODE REVIEW

File: components/Homepage/HomepageSplitHero.tsx [hunk 1]
```diff
@@ -121,10 +121,7 @@ export const HomepageSplitHero = () => {
             For combating the climate crisis. For reducing inequality. For embedding ethics in all we do.
           </Paragraph>
           <Paragraph size={1} weight="normal" leading="snug" className={styles.introParagraph}>
-            Welcome to <em>Momentum</em>, telling the stories of Stanford’s impact.
-          </Paragraph>
-          <Paragraph size={1} weight="normal" leading="snug" className={styles.introParagraph}>
-            Come discover the work underway in four key thematic areas.
+            Join us as we tell the stories of Stanford’s impact.
```
Comment (yvonnetangsu) [inline components/Homepage/HomepageSplitHero.tsx:124]: Updated homepage intro 12/14

File: components/RichText.tsx [hunk 2]
```diff
@@ -93,9 +93,12 @@ export const RichText = ({
...
         const { level } = props;
-        // All heading sizes are type-1 if using the "card" type WYSIWYG
-        // For regular main content WYSIWYG, this gets you type-4 for h2, type-3 for h3, etc.
-        const headingSize = type === 'default' ? 6 - level : 1;
+        /**
+         * All heading sizes are type-1 if using the "card" type WYSIWYG
+         * For regular main content WYSIWYG, this gets you type-3 for h2, type-2 for h3, type-1 for h4
+         * but h5 and h6 would be type-0 (the minimum font size)
+         */
+        const headingSize = type === 'default' ? Math.max(5 - level, 0) : 1;
```
Comment (yvonnetangsu) [inline components/RichText.tsx:101]: Stepping down WYSIWYG heading sizes by 1 modular type. This keeps the minimum at type-0 (same size as regular paragraph)

File: utilities/config.ts [hunk 3]
```diff
@@ -4,7 +4,7 @@
 export const config = {
   siteUrlProd: 'https://momentum.stanford.edu',
   siteTitle: 'Stanford Momentum',
-  siteDescription: 'TBD', // TODO: Add generic site description as fallback for search engine if Storyblok SEO description is not provided
+  siteDescription: 'Stories of Stanford’s impact, fueled by philanthropy.',
```
Comment (yvonnetangsu) [inline utilities/config.ts:7]: Add site description. If a page has no description, it will use this one.

**Comment by yvonnetangsu** [comment]
@sherakama I'm going to merge this it so it fixes the broken builds due to the caption change to rich text. Please review retro 😄

**Review by sherakama — COMMENTED** [review]
👍

**Pull request merged**